Course.js

```diff
@@ -134,3 +135,26 @@ COURSES = [
 	new Course("AE 4.2", "Global cultural diversity", [], [], [1,1,1], 3),
 	new Course("AE 5.1", "Social Responsibility and Ethics", [], [], [1,1,1], 3)
 ];
+
+//Indices: 0 = CS; 1 = CoE; 2 = EE; 3 = IC-A; 4 = IC-B; 5 = IC-C; 6 = IC-G; 7 = IC-J; 8 = IC-P; 9 = Exceptions
+ULE = [
```

Comment:
If these ULEs are major-specific, they should ideally be stored within the Major class instead of being assigned these arbitrary array indicies. It would be fine if the exceptions still work this way though.